.gitmodules

```diff
@@ -14,3 +14,7 @@
 	path = Common/MU_OEM_SAMPLE
 	url = https://github.com/microsoft/mu_oem_sample.git
 	branch = release/202008
+[submodule "Common/PRM"]
```

Comment:
I don't love adding new submodules. Do we have an ETA or roadmap to moving this into a mu repo?

Reply:
Not in the near term since it's on a path for interception in edk2 and some other PRM users prefer to pull PrmPkg internally though edk2-staging. Once it lands in edk2, we will be able to pull it from there like other packages.